lib/iframe-browser.js

```diff
@@ -17,6 +17,7 @@ define(function (require, exports, module) {
     var detachedWindow;
     var isReload;
     var PostMessageTransport = require("lib/PostMessageTransport");
+    var Compatibility = require("lib/compatibility");
```

Comment:
I tend to agree, except that Brackets seems to capitalize ModuleNames when requiring.  I'm just following that.